LandSandBoat/server
# [SMN] Correct Garuda animations & Blood Pact abilities

**_I affirm:_**

## What does this pull request do?



Updates all of Garudas animations to reflect new pet_skills sql table
Compared all Blood pact msg's to retail captures
Removes Garuda's animation shifting from core
Added SQL updates for future Garuda Blood Pact abilities

## Steps to test these changes


Use Garuda's BP Ward to see proper MP consumption and messages displayed
Use Garuda's BP Rage to see proper MP consumption and messages displayed
